examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+#
+# This is an example of plotting waveforms read from an NI-SCOPE device through gRPC
```

Comment:
I think this description is taken from scope example. But this has to describe the current example. Can you discuss with @harsha-bhushan  about what to provide as one line description for this example or can we retain the existing one itself?